ada_2_fhir-r4/cio/2.0.0/beschikbaarstellen_contraindicatie/fhir_instance/mci-MCI-zwangerschap-1.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Flag xmlns="http://hl7.org/fhir">
+   <id value="mci-MCI-zwangerschap-1"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/cio-MedicationContraIndication"/>
+   </meta>
+   <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationContraIndication.RegistrationDateTime">
+      <valueDateTime value="${DATE, T, Y, -30}T08:54:38+02:00"/>
+   </extension>
+   <identifier>
+      <system value="urn:oid:2.16.840.1.113883.2.4.3.11.999.26.1.2"/>
+      <value value="MCI-zwangerschap-1"/>
+   </identifier>
+   <status value="active"/>
```

Comment:
realistiche situatie? Met een zwangerschap die 30 jaar geleden is begonnen?

Reply:
Goed punt! Hier had ik ook al mijn vraagtekens bij, en CIO-89 is aangemaakt om de precieze betekenis achter het statusveld in de dataset van CiO te bepalen. Het lijkt erop dat de IA'ers heel anders tegen de status aankijken, en een conclusie zou zelfs kunnen zijn dat de mapping van de MedicatieContraIndicatieStatus uit de dataset op het element `Flag.status` in FHIR semantisch niet correct is (en dan kan dat in de volgende (beta)release aangepast worden).

Reply:
Uitgangspunt vanuit IA is in ieder geval: reeds bestaande instanties worden nóóit meer aangepast. Als je dit dan weer naar FHIR vertaalt, krijg je rare dingen als je alle _current_ `Flag`s ophaalt.